kolibri/plugins/__init__.py

```diff
@@ -221,6 +220,13 @@ def class_module_path(self):
     def module_path(self):
         return self.class_module_path()
 
+    def name(self, lang):
+        """
+        A function to return a name for the plugin.
+        Accepts a lang argument to allow for internationalization of the name.
```

Comment:
Is the i18n baked in via something not clear here - or just not yet implemented?

Reply:
In specific plugins overriding this method, they would use the lang parameter to activate translation then use regular Django mechanisms to translate.

The reason for not enforcing anything more specific is that in the case of plugins external to Kolibri, they won't be able to use Django translation mechanisms, as the locale files will not be discoverable by Django until the plugin has been enabled, so they would need to use basic Python i18n machinery to offer a translated name.

Reply:
To make this clearer, I added in placeholder functions in all the relevant plugins.